homeassistant/components/linkplay/strings.json

```diff
@@ -22,5 +22,17 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "services": {
+    "preset": {
+      "name": "Preset",
```

Comment:
```suggestion
      "name": "Play preset",
```

Reply:
Maybe consider renaming the service from `preset` to `play_preset` as `linkplay.preset` doesn't sound like an action, rather an attribute